rpcs3/Emu/RSX/GSRender.cpp

```diff
@@ -39,6 +39,7 @@ void GSRender::on_exit()
 
 	if (m_frame)
 	{
+		m_frame->hide();
```

Comment:
Why is this necessary?

Reply:
Better user experience, especially for savestates to show the progress dialog.